include/net/ptp_time.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Public functions for the Precision Time Protocol time specification.
+ *
+ */
+
+#ifndef __PTP_TIME_H
+#define __PTP_TIME_H
+
+/**
+ * @brief Precision Time Protocol time specification
+ * @defgroup ptp_time PTP time
+ * @ingroup networking
+ * @{
+ */
+
+#include <net/net_core.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Precision Time Protocol Timestamp format.
+ *
+ * This structure represents a timestamp according
+ * to the Precision Time Protocol standard.
+ *
+ * Seconds are encoded on a 48 bits unsigned integer.
```

Comment:
> Seconds are encoded on a 48 bits unsigned integer.

Having not read PTP spec, I can't understand this comment. "Seconds are encoded as a 48 bits unsigned integer." or "Seconds are encoded in a 48 bits unsigned integer." would be clearer, but still not fully. Can this structure carry *either* seconds or nanoseconds? If so, why not say that explicitly in the description?

Reply:
> Can this structure carry either seconds or nanoseconds? If so, why not say that explicitly in the description?

Well, nevermind, got confused by indentation, they're both there. The comment regarding using "as" instead of "on" still holds ;-).